File: Problem-6/main.go
package main

import "fmt"

func main()  {
    var (
        sqr_sum = 0
        sum_sqr = 0
        result = 0
    )

    for i := 0; i < 100; i++ {
        sqr_sum = sqr_sum + (i * i)
        sum_sqr = sum_sqr + i
    }

    result = (sum_sqr * sum_sqr) - sqr_sum
    fmt.Println(result)
}

File: Problem-6/main.ts
let sqr_sum: number = 0;
let sum_sqr: number = 0;

for (var i = 0; i > 100; i++) {
   sqr_sum = sqr_sum + i * i;
   sum_sqr = sum_sqr + i;
}

let answer: number = sum_sqr * sum_sqr - sqr_sum;
console.log(answer);
